utils.go

```diff
@@ -0,0 +1,18 @@
+package main
```

Comment:
@BenjaminTrapani Nope. My comment said the package name, "main" in this case, was wrong.